cardano-tx-generator/src/Cardano/Benchmarking/GeneratorTx.hs

```diff
@@ -700,8 +712,9 @@ runBenchmark benchTracer
                           fundsWithGenesisMoney
                           explorerAPIEndpoint
 
-  -- sleep for 20 s; subsequent txs enter new block
-  liftIO $ threadDelay (20*1000*1000)
+  liftIO . traceWith benchTracer . TraceBenchTxSubDebug
+    $ "******* Tx generator: waiting " ++ show initCooldown ++ "s *******"
+  liftIO $ threadDelay (initCooldown*1000*1000)
 
   liftIO . traceWith benchTracer . TraceBenchTxSubDebug
```

Comment:
one could also change the context name to indicate in which phase the message appeared.